template/user.deactivation/body.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>
+      Notice: Your CodeReady Toolchain / OpenShift.io free account is inactive
+    </title>
+    <style>
+      a:hover {
+        text-decoration: underline !important;
+      }
+      p {
+        text-align: left;
+        margin: 30px 0;
+      }
+    </style>
+  </head>
+
+  <body
+    style="
+        padding: 10px;
+        padding: 0;
+        background-color: #f9f9f9;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 15px;
+        font-weight: lighter;
+        line-height: 1.2;"
+  >
+    <div
+      style="
+        min-height: 300px;
+        max-width: 750px;
+        margin: 0 auto;
+        padding: 20px;
+        border: 1px solid #d7d7d7;
+        border-radius: 4px;
+        background-color: #fff;
+        box-shadow: 0 2px 4px #d7d7d7;"
+    >
+      <img
+        src="{{.webURL}}/static/codeready_toolchain_dark.png"
+        style="max-width: 75%"
+        alt="CODEREADY TOOLCHAIN"
+      />
+      <div
+        style="
+            margin: 15px auto 0;
+            border: 1px solid #d7d7d7;"
+      ></div>
+
+      <p>
+        You are receiving this email because you have an OpenShift.io Online
+        free account ({{ .custom.userEmail }}) and your action is required to
+        keep it active.
+      </p>
+
+      <p>
+        Login into the system by {{ .custom.expiryDate }} in order to continue
+        using OpenShift.io uninterrupted.
+      </p>
+
+      <p>
+        <a href="https://openshift.io">OpenShift.io</a> |
+        <a href="https://che.openshift.io">Che.OpenShift.io</a>
+      </p>
+
+      <p>
+        If you don't have time to use OpenShift Online right now, that's ok too.
+        We will remove your access, delete the data and allocate resources to
+        another user waiting to use the platform.
+      </p>
```

Comment:
In this, `If you don't have time to use OpenShift Online right now`
It should be "OpenShift.io Online" instead of "OpenShift Online" .. right ??

Reply:
the text comes from an external document, but I can ask for confirmation, indeed.

Reply:
actually, the term `OpenShift.io Online` is already used above, so I made the change you suggested in ff20314